SpliceAIWeb/Dockerfile

```diff
@@ -12,5 +18,13 @@ RUN pip install --no-cache-dir -U pip \
 COPY . /app
 WORKDIR /app
 
-EXPOSE 5001
-CMD ["gunicorn", "-b 0.0.0.0:5001", "-t 1200", "-w 4", "spliceai_web.app:app"]
+RUN chmod +x /app/gcsfuse_run.sh
+
+EXPOSE 8080
+
+# Use tini to manage zombie processes and signal forwarding
+# https://github.com/krallin/tini
+ENTRYPOINT ["/usr/bin/tini", "--"] 
```

Comment:
Why use both entrypoint and CMD. Is this so default execution runs the gcsfuse_run.sh, but you can overwrite another executable that always runs through tini? Kind of cool if so, but would be good to add a comment - I've never seen ENTRYPOINT / CMD used at the same time.